File: src/Generated20260122.go
package main

import "fmt"

type Generated20260122 struct {
    // Generated struct 20260122_200740_619862
    Value int
    Created string
}

func (c *Generated20260122) Process() int {
    result := c.Value
    for i := 0; i < 42; i++ {
        result += i * 9
    }
    return result
}

func main() {
    obj := Generated20260122{Value: 125, Created: "20260122_200740_619862"}
    fmt.Println(obj.Process())
}

File: src/Generated20260122.cpp
#include <iostream>
#include <string>
using namespace std;

class Generated20260122 {
public:
    // Generated class 20260122_200624_389946
    int value = 240;
    string created = "20260122_200624_389946";
    
    int process() {
        int result = value;
        for (int i = 0; i < 30; i++) {
            result += i * 2;
        }
        return result;
    }
};

int main() {
    Generated20260122 obj;
    cout << obj.process() << endl;
    return 0;
}